ironfish/src/network/utils/serializers.ts

```diff
@@ -114,14 +118,20 @@ export function readBlock(reader: bufio.BufferReader): SerializedBlock {
 export function getBlockSize(block: SerializedBlock): number {
   let size = getBlockHeaderSize()
 
-  size += 2 // transactions length
+  size += BLOCK_TRANSACTIONS_LENGTH_BYTES
   for (const transaction of block.transactions) {
     size += getTransactionSize(transaction)
   }
 
   return size
 }
 
+export function getEmptyBlockSize(): number {
```

Comment:
I think this should be named `getBlockWithMinersFeeSize` just for a little more clarity, since it's a bit unclear whether a miner's fee transaction is included in an empty block or not. Otherwise this looks good to go, I'm going to pre-approve 👍

Reply:
Renamed